src/translations/enTranslations.json

```
@@ -482,12 +483,14 @@
     "my_profile": {
       "heading": {
         "affiliations": "Affiliations",
+        "personalia": "Personalia",
         "roles": "Roles",
         "user_info": "User information"
       },
       "id": "ID",
       "orcid": {
         "connect_orcid": "Connect/create ORCID iD",
+        "delete_orcid": "Remove ORCID-connection",
```

Bindestrek en vel en typisk norsk sak
```suggestion
        "delete_orcid": "Remove ORCID connection",
```